release-tools/MKRELEASE.md

```diff
@@ -3,76 +3,87 @@
+*WARNING*  You are advised to run this command only in a fresh worktree which
+contains no uncommitted changes or untracked files, because unless you specify
+`--no-clean`, the release process will run `git clean -x -d -f`, which will
+delete ALL untracked files from the worktree.
+See the [README](README.md#prepare-the-worktrees) file for a suggestion how to
+prepare the worktrees.
 
-From the branch directory if you run:
+Change into the worktree and run:
 
-        perl /path/to/mkrelease.pl --reviewer=name
+        mkrelease.pl --reviewer=name
 
-It should perform all of the above steps and commit changes locally. You can
-then sanity check these with `git log` before pushing them to the public repo.
+The script should perform all of the above steps and commit the changes locally.
+You can then sanity check these with `git log` before pushing them to the public
+repository.
 
 ## Environment Variables
 
-- OPENSSL<br>
-  Path to openssl utility to use. Default is `openssl`.
+The following is a list of environment variables used by the script, together with
+their default values in square brackets:
 
-- OPENSSL_TAR<br>
-  The tar command to use when creating the tarball. Default is `tar`.
+        OPENSSL                    [openssl]
 
-- OPENSSL_GPG<br>
-  The gpg command to use when signing a tarball or announcement.
-  The default is `gpg` which will use gpg with the default key.
-  If you wish to use a different, key set OPENSSL_GPG to include
-  appropriate options.
+Path to openssl utility to use. Default is
 
-- OPENSSL_GPG_TAR<br>
-  Command to use to sign a tarball.
-  The default is: `$OPENSSL_GPG --use-agent -sba`
+        OPENSSL_TAR                [tar]
 
-- OPENSSL_GPG_ANNOUNCE<br>
-  The command to use to sign a tarball.
-  The default is: `$OPENSSL_GPG --use-agent -sta --clearsign`
+The tar command to use when creating the tarball. Default is `tar`.
 
-- OPENSSL_SCP<br>
-  The command to use to upload files.
-  The default is `scp`
+        OPENSSL_GPG                [gpg]
 
-- OPENSSL_SCP_HOST<br>
-  The host (and optional username) needed to upload files.
-  The default is `dev.openssl.org`, but you might want to change this to
-  `username@dev.openssl.org`
+The gpg command to use when signing a tarball or announcement.
+The default is `gpg` which will use gpg with the default key.
+If you wish to use a different, key set OPENSSL_GPG to include
+appropriate options.
 
-- OPENSSL_SCP_DIR<br>
-  The directory to upload files to.
-  Normally this wont be changed from the default which is
-  `$OPENSSL_SCP_HOST:~openssl/dist/new`
-  This is a holding area on dev.openssl.org where distributions are uploaded
-  temporarily before being moved to the web and ftp directories.
+        OPENSSL_GPG_TAR            [$OPENSSL_GPG --use-agent -sba]
+
+Command to use to sign a tarball.
+
+        OPENSSL_GPG_ANNOUNCE       [$OPENSSL_GPG --use-agent -sta --clearsign]
+
+The command to use to sign a tarball.
```

Comment:
What is the difference between `OPENSSL_GPG_TAR` and `OPENSSL_GPG_ANNOUNCE`, are they both used for signing a tarball?